### DISCUSSION TITLE
Skip Empty 'impl' Block Generation in 'sea-orm-cli generate entity' Command

### DISCUSSION THREAD

**Discussion by kaplanelad** [SeaQL/sea-orm#1878] [Ideas]
I propose the addition of a flag to the CLI command sea-orm-cli generate entity that allows for skipping the generation of an empty impl block, specifically impl ActiveModelBehavior for ActiveModel {}.

```
sea-orm-cli generate entity --skip-generate-active-block_behaivior [list of tables]
```

For instance, when I intend to hash a user's password before inserting a new record, I would like to use the before_save implementation. Currently, this involves the following steps:

1. Adding the necessary logic directly to the generated file. However, this code gets deleted when the generate entity command is rerun.

2. Attempting to place the before_save logic in a different location but getting compilation error: "conflicting implementation" error.

The proposed flag would streamline this process and prevent the deletion of custom code during regeneration.

**Comment by kaplanelad** [comment]
f it sounds acceptable, I am willing to take on this task and implement the feature.

**Comment by waynr** [comment]
@kaplanelad FYI I went ahead and did this, then came across your proposal: https://github.com/SeaQL/sea-orm/pull/1947

Better to ask forgiveness than permission :). My PR takes a different approach semantically for the flag but has the same effect (leave the default behavior as it is today but provide a way for the user to skip it). I might rename the CLI option in my PR to match what you've proposed.
